### DISCUSSION TITLE
Audit logs entry as clear event even if we click on the tasks.

### DISCUSSION THREAD

**Discussion by sktemkar** [apache/airflow#35993] [Q&A]
Whenever I click on the task, there is an entry added in the audit logs as task being cleared. This is misleading.
I want the report from the audit logs to check who cleared the tasks from the airflow ui. 

Please let me know how to remove this logs. airflow versions - 2.6.1, 2.7.1
![Capture](https://github.com/apache/airflow/assets/17265855/c84af74f-f04e-4336-9b38-7f29cbc473f9)

**Comment by potiuk** [comment]
If you think this is an error, then you can report it as issue. It would be great if you show exactly what you do and what exactly you mean by "click on the task" - not here - but in the description of the bug report.  Ideally record a video showing what you do to avoid ambigutities.

Then - if this turns out to be a bug, we encourage our users to attempt to fix it and submit a PR, otherwise we can diagnose it and mark it as good first issue and it's likely someone will pick it up and fix it in the next version.
